test/TesterInternal/Deployment/AzureSiloTests.cs

```diff
@@ -90,4 +96,13 @@ public IPEndPoint GetIPEndpoint(string endpointName)
 
         public void UnsubscribeFromStoppingNotification(object handlerObject, EventHandler<object> handler) { }
     }
+
+    public class Startup : IStartup
+    {
+        public IServiceProvider ConfigureServices(IServiceCollection svcCollection)
+        {
+            OrleansInternalServices.RegisterSystemTypes(svcCollection);
+            return svcCollection.BuildServiceProvider();
+        }
+    }
```

Comment:
This is a pattern borrowed from ASP.Net startUp class pattern : https://docs.asp.net/en/latest/fundamentals/startup.html
TODO here to put configuration and initialization in this class, for example, init an AzureSilo.